tasks.py

```diff
@@ -545,7 +549,7 @@ def let_through(self, node):
             return self.construct_mapping(node)
 
     YamlUnity3dTag.add_constructor(
-        u"tag:unity3d.com,2011:47", YamlUnity3dTag.let_through
+        "tag:unity3d.com,2011:47", YamlUnity3dTag.let_through
```

Comment:
I'm not sure what the `u` prefix does here, I assume removing this is fine?

Reply:
That is a relic from Python 2.7 support.  In Python2.7 prefixing a string with `u` creates a unicode string.  This isn't necessary with Python 3.